### PR TITLE
ci: add unit testing for kotlin

### PR DESCRIPTION
**Pull request by buildbreaker**
Add unit testing CI for kotlin tests in `//library/kotlin/test/...`

Signed-off-by: Alan Chiu <achiu@lyft.com>

For an explanation of how to fill out the fields, please see the relevant section
in [PULL_REQUESTS.md](https://github.com/envoyproxy/envoy/blob/master/PULL_REQUESTS.md)

Description: CI for kotlin unit testing
Risk Level: low
Testing: soon
Docs Changes: none
Release Notes: none
[Optional Fixes #Issue]
[Optional Deprecated:]

### CODE REVIEW

**Comment by buildbreaker** [comment]
@junr03 Ahhh...it was a moved file. Fixed!

File: azure-pipelines.yml [hunk 1]
```diff
@@ -152,6 +152,25 @@ stages:
...
+            submodules: true
+          - script: ./ci/linux_ci_setup.sh
+            displayName: 'Install dependencies'
+          - script: |
+              export PATH=/usr/lib/llvm-8/bin:$PATH
+              export CC=clang
+              export CXX=clang++
+            displayName: 'Set up clang toolchain'
+          - script: |
+              ./bazelw query 'tests(//library/kotlin/...)' | xargs ./bazelw test --test_output=all
```
Comment (junr03) [inline azure-pipelines.yml]: why do we need to run the query. Can't we just do `./bazelw test //library/kotlin/test/...`?

Comment (buildbreaker) [reply azure-pipelines.yml]: 😂

Comment (goaway) [reply azure-pipelines.yml]: this is Jose impersonating Mike. Even with the flag I would still use the test path for explicitness.

**Pull request merged**